user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Other/Visual_Overview_component.md

```diff
@@ -26,6 +26,7 @@ To configure this component:
 >
 > - Spectrum components are currently not yet supported in visual overviews within dashboards or low-code apps.
 > - Quick filters are supported for table parameters in visual overview components. See [Using quick filters](xref:Using_quick_filters).
+> - In versions prior to DataMiner 10.3.0 [CU19]/10.4.0 [CU7]/10.4.10<!--RN 40497-->, when you have multiple components that show the same visual overview with user context (e.g. a visual overview with card variables), the contexts of those components are shared. For example, if you click a button on one component, the same action will be mirrored on the other component.
```

Comment:
This change is only for feature release 10.4.10, the release note is inaccurate because of differences between normal Cube and this mobile version

Reply:
```suggestion
> - In versions prior to DataMiner Web 10.4.10<!--RN 40497-->, when you have multiple components that show the same visual overview with user context (e.g. a visual overview with card variables), the contexts of those components are shared. For example, if you click a button on one component, the same action will be mirrored on the other component.
```

Reply:
It would be the best to have it as just 10.4.10, the code for mobile visual overview is not provided with the web only upgrades, only with full upgrades so only users who will install the 10.4.10 full upgrade and subsequent upgrades will get this feature

Reply:
OK! I changed it back to "DataMiner 10.4.10". Thanks for the feedback.